homeassistant/components/lutron_caseta.py

```diff
@@ -41,9 +41,7 @@ def setup(hass, base_config):
 
     config = base_config.get(DOMAIN)
     hass.data[LUTRON_CASETA_SMARTBRIDGE] = Smartbridge(
-        hostname=config[CONF_HOST],
-        username=config[CONF_USERNAME],
-        password=config[CONF_PASSWORD]
```

Comment:
The user name and password are no longer required.  The lutron ssh integration works with a hardcoded key/user that cannot be changed.  I made them optional​ just in case some folks already had them configured.  Should I just removed them?  The allow extra should allow already configured username and password to continue working.  Basically they will be ignored.